Source/WebCore/platform/graphics/PlatformColorSpace.h

```diff
@@ -39,6 +41,11 @@ namespace WebCore {
 using PlatformColorSpace = RetainPtr<CGColorSpaceRef>;
 using PlatformColorSpaceValue = CGColorSpaceRef;
 
+#elif USE(SKIA)
+
+using PlatformColorSpace = sk_sp<SkColorSpace>;
+using PlatformColorSpaceValue = sk_sp<SkColorSpace>;
```

Comment:
I guess this would be `SkColorSpace*`, or do we want to use the smart pointer as value too?

Reply:
Actually we must use `sk_sp<SkColorSpace>` as in many cases we need to use `sk_sp<SkColorSpace>` using `PlatformColorSpaceValue`